daprdocs/content/en/concepts/dapr-services/placement.md

```diff
@@ -19,6 +19,10 @@ The placement service is deployed as part of `dapr init -k`, or via the Dapr Hel
 
 There is an [HTTP API `/placement/state` for placement service]({{< ref placement_api.md >}}) that exposes placement table information. The API is exposed on the sidecar on the same port as the healthz. This is an unauthenticated endpoint, and is disabled by default. You need to set `DAPR_PLACEMENT_METADATA_ENABLED` environment or `metadata-enabled` command line args to true to enable it. If you are using helm you just need to set `dapr_placement.metadataEnabled` to true.
 
+{{% alert title="Important" color="warning" %}}
+When running placement in [multi-tenant mode]({{< ref namespaced-actors.md >}}), disable the `metadata-enabled` command line args to prevent different namespaces from seeing each other's data.
+{{% /alert %}}
+
 ### Usecase:
 The placement table API can be used for retrieving the current placement table, which contains all the actors registered. This can be helpful for debugging and allows tools to extract and present information about actors.

```

Comment:
```suggestion
The placement table API can be used to retrieve the current placement table, which contains all the actors registered across all namespaces. This can be helpful for debugging and allows tools to extract and present information about actors.

```